D4RL (Mujoco).rst

```diff
@@ -0,0 +1,343 @@
+    # Extract the downloaded mujoco200 directory into ~/.mujoco/mujoco200.
+    unzip mujoco200_linux.zip && mkdir -p ~/.mujoco && mv mujoco200_linux ~/.mujoco/mujoco200
+    # Download unlocked activation key.
+    wget https://www.roboti.us/file/mjkey.txt -O  ~/.mujoco/mjkey.txt 
+    # Install and use mujoco-py
+    pip install gym
+    pip install -U 'mujoco-py<2.1'
+
+    # Installation for macOS
+    # Download the MuJoCo version 2.0 binaries for OSX.
+    wget https://www.roboti.us/download/mujoco200_macos.zip
+    # Extract the downloaded mujoco200 directory into ~/.mujoco/mujoco200.
+    tar xvf mujoco200-macos-x86_64.tar.gz && mkdir -p ~/.mujoco && mv mujoco200_macos ~/.mujoco/mujoco200
+    # Download unlocked activation key.
+    wget https://www.roboti.us/file/mjkey.txt -O  ~/.mujoco/mjkey.txt 
+    # Install and use mujoco-py
+    pip install gym
+    pip install -U 'mujoco-py<2.1'
+
+
+Verifying the Installation
+--------------------------------
+
+Once the installation is complete, you can verify that the installation was successful by running the following command from the Python command line.
+
+.. code:: python
+
+    import gym
+    import d4rl # Import required to register environments
+
+    # Create the environment
+    env = gym.make('maze2d-umaze-v1')
+
+    # d4rl abides by the OpenAI gym interface
+    env.reset()
+    env.step(env.action_space.sample())
+
+    # Each task is associated with a dataset
+    # dataset contains observations, actions, rewards, terminals, and infos
+    dataset = env.get_dataset()
+    print(dataset['observations']) # An N x dim_observation Numpy array of observations
+
+    # Alternatively, use d4rl.qlearning_dataset which
+    # also adds next_observations.
+    dataset = d4rl.qlearning_dataset(env)
+
+Mirroring
+-------------
+
+The DI-engine is ready with images of the framework itself, which can be obtained by \ ``docker pull opendilab/ding:nightly-mujoco``, or by visiting \ `docker
+hub <https://hub.docker.com/r/opendilab/ding>`__\ for more mirrors
+
+
+GYM-MUJOCO'S SPACE BEFORE TRANSFORMATION (ORIGINAL ENVIRONMENT)
+=====================================================================
+
+
+Observation Space
+---------------------
+
+-  The vector consists of physical information(3D position, orientation, and joint angles etc.), and the specific size is \ ``(N, )``\, which is determined by the environment. The data type is \ ``float64``
+-  `Fujimoto <https://github.com/opendilab/DI-engine/blob/main/dizoo/d4rl/entry/d4rl_cql_main.py>`__ mentions that doing obs norm for the d4rl dataset will improve the stability of offline training.
+
+
+Action Space
+----------------
+
+-  The vector consists of physical information(torque etc.), which is often the continuous action space.（N varies with specific subenvironments), the data type is \ ``float32``\ ，需要传入 np 数组（例如动作为\ ``array([-0.9266078 , -0.4958926 ,  0.46242517], dtype=float32)``\ ）
```

Comment:
1. "..., which is often a continuous action space with N dimension (N varies...."
2. translate the chinese here